docs/monitoring.md

```diff
@@ -0,0 +1,34 @@
+# IDR Monitoring
+
+## Metrics
+
+Day to day performance of the IDR can be visualised using Grafana, which is accessible using an anonymous account on idr-management:
+
+    ssh idr-proxy -L 3000:management:3000
+
+Open http://localhost:3000/.
+The dashboards of interest are accessed from the `Home` menu:
+- IDR PostgreSQL: The numbers of database rows and locks in use
+- IDR sessions: Numbers of OMERO sessions and web requests. Note that quantiles are calculated over a fixed window, and will be very inaccurate if the number of requests is low.
+- IDR vertical: CPU, active memory, load and network usage for all servers.
+
+### Prometheus
+
+If you want to query the raw metrics directly you can connect to prometheus:
+
+    ssh idr-proxy -L 9090:management:9090
```

Comment:
2018-04-19 13:36 LS27172:~ kenny$ ssh idr-proxy -L 9090:management:9090
ssh: Could not resolve hostname idr-proxy: nodename nor servname provided, or not known
2018-04-19 13:36 LS27172:~ kenny$ host idr-proxy
Host idr-proxy not found: 3(NXDOMAIN)

Reply:
`idr-proxy` is a placeholder since the actual hostname will depend on where it's deployed